examples/social/tests/cross_chain.rs

```diff
@@ -48,14 +48,24 @@ async fn test_cross_chain_posting() {
     chain1.handle_received_effects().await;
 
     // Querying the own posts
-    let query_string = "query { ownPosts(start: 0, end:1) { timestamp, text } }";
-    let value: serde_json::Value = chain2.query(application_id, query_string).await;
-    let value = value["data"]["ownPosts"][0]["text"].clone();
+    let query_string = "{ ownPosts(start: 0, end:1) { timestamp, text } }";
+    let response = chain2
+        .query(application_id, query_string.into())
+        .await
+        .data
+        .into_json()
+        .expect("Unexpected non-JSON query response");
+    let value = response["ownPosts"][0]["text"].clone();
     assert_eq!(value, "Linera is the new Mastodon".to_string());
 
     // Now handling the received messages
-    let query_string = "query { receivedPostsKeys { timestamp, author, index } }";
-    let value: serde_json::Value = chain1.query(application_id, query_string).await;
-    let author = value["data"]["receivedPostsKeys"][0]["author"].clone();
+    let query_string = "{ receivedPostsKeys { timestamp, author, index } }";
+    let response = chain1
+        .query(application_id, query_string.into())
```

Comment:
Since most of our `QueryResponse`s are GraphQL responses, maybe we should have a `query_json` helper method that returns the JSON, so we don't have to repeat `.data.into_json().expect("Unexpected non-JSON query response")` everywhere.

Reply:
Good idea! But I made it `graphql_query` instead, because I think it explains the constraint better :+1: